test/common/markdown/markdown-link.tests.ps1

```diff
@@ -0,0 +1,93 @@
+﻿Describe "Verify Markdown Links" {
+    BeforeAll {
+        # Cleanup jobs for reliability
+        Get-Job | stop-job
+        get-job | remove-job
+    }
+
+    AfterAll {
+        # Cleanup jobs to leave the process the same
+        Write-Verbose -verbose "cleaning jobs ..."
+        Get-Job | stop-job
+        get-job | remove-job
+    }
+
+    $groups = Get-ChildItem -Path "$PSScriptRoot\..\..\..\*.md" -Recurse | Group-Object -Property directory
+
+    $jobs = @{}
+    # start all link verification in parallel
+    Write-Verbose -verbose "starting jobs for performance ..."
+    Foreach($group in $groups)
+    {
+        $result = markdown-link-check $file 2>&1
+        $job = start-job {
```

Comment:
You might consider using ThreadJobs (https://www.powershellgallery.com/packages/ThreadJob/1.1.2) since they are lighter weight.